apps/sms/views/conversation/js/conversation.js

```diff
@@ -80,6 +80,11 @@ var ConversationView = {
   showErrorInFailedEvent: '',
   previousSegment: 0,
 
+  /**
+   * Set to true after init is run.
+   */
+  _isReady: false,
```

Comment:
Yeah something like this, I was thinking more about view behavior proxies/mocks with promise-based "init" method, so Navigation calls ```ViewBehavior.init()``` that loads view behavior object if needed - but will see later.